test/test_xpu.py

```diff
@@ -235,6 +236,21 @@ def test_events(self):
         stream.record_event(event)
         event.synchronize()
         self.assertTrue(event.query())
+        start_event = torch.xpu.Event(enable_timing=True)
+        end_event = torch.xpu.Event(enable_timing=True)
+        stream.record_event(start_event)
+        time.sleep(0.1)
+        stream.record_event(end_event)
+        torch.xpu.synchronize()
+        if int(torch.version.xpu) >= 20250000:
```

Comment:
It's not related to this PR. But why torch.version.xpu is going to tell the version of compiler?

Reply:
Aligning to cuda version, like 11.8, 12.4...